### ISSUE TITLE
Disc group actions

### ISSUE THREAD

**Issue by OxygenCobalt** [OxygenCobalt/Auxio#406]
Add functionality to play individual disc groups in the album view. Not quite sure how this would work exactly, but it should be a nice QoL improvement.

**Comment by KraXen72** [comment]
i think just treating it as a songitem where you can click to play it, or hold to select all of the disc to play it next or add it to queue could work. however, even currently, the disc group separator blends right in with the songs - unless this is intended behavior, i would suggest somehow differentiating it because it gives the illusion that there are more songs in the album or that it's not split into discs at all.

**Comment by OxygenCobalt** [comment]
Yeah, I agree that currently the disc separator is currently non-ideal. The issue is that I can't just make it another header since:
- It would create a redundant header given the presence of the "songs" header. If I removed that header, then I wouldn't know where to put the sort button.
- The disc item is two-lined, so the normal header layout does not work anyway.

As for the behavior, I'm thinking about just making a disc long-click map to selecting all of the songs in a disc group. It would enable all that you described, but it would be somewhat hidden functionality since I don't have a better layout to communicate it.
